{{ cookiecutter.dir_name }}/src/android/PythonActivity.java

```diff
@@ -3,9 +3,10 @@
 import android.app.Activity;
 import android.os.Bundle;
 import android.util.Log;
+import android.support.v7.app.AppCompatActivity;
 
 
-public class PythonActivity extends Activity {
+public class PythonActivity extends AppCompatActivity {
```

Comment:
I've simplified the theme usage, since we're going to be using code to define this stuff by preference.